usage/gpio-plus-and-raspi2/README.md

```diff
@@ -1,89 +0,0 @@
-# GPIO: Models A+, B+, Raspberry Pi 2 B and Raspberry Pi 3 B
```

Comment:
OK, for this particular case, we would prefer a placeholder page with a link back to the new combined page. Thanks.